### PR TITLE
fix(null-and-undefined): Describe `find` cases and give `update` a headline

### PR DESCRIPTION
**Pull request by janpio**
Part of untangling https://github.com/prisma/prisma/issues/17723

### CODE REVIEW

**Comment by sabinadams** [comment]
@janpio Made some pretty extensive changes to the page to make it flow a bit more and simplify the explanations.

Let me know what you think 🙂

**Comment by janpio** [comment]
I think this is ready for a review @jharrell @ruheni - does not fix the page, but makes it better in small ways to make more structured.

**Review by jharrell — APPROVED** [review]

**Comment by janpio** [comment]
This need another set of proper reviews, I can honestly not make sense of the changes myself.

File: content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx [hunk 1]
```diff
@@ -12,90 +12,235 @@ Prisma Client differentiates between `null` and `undefined`:
...
+| id  | name    | email             |
+| --- | ------- | ----------------- |
+| 1   | Nikolas | nikolas@gmail.com |
+| 2   | Martin  | martin@gmail.com  |
+| 3   | _empty_ | sabin@gmail.com   |
+| 4   | Tyler   | tyler@gmail.com   |
+
+</TopBlock>
+
+<!-- TODO Simple example with `findMany` and `findFirst` that shows how a query with `foo: undefined` (e.g. https://github.com/prisma/prisma/issues/17723) returns everything or any first entry as it is filtered out of the query -->
```
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx:32]: Is this a todo for later or for this PR?

**Review by jharrell — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx (hunk 1 above)
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx:32]: No idea, you'll have to figure that out from the context.

**Pull request merged**